apps/nextra/public/scripts/install_cli.py

```diff
@@ -483,33 +496,127 @@ def get_target(self):
             self._write("")
             sys.exit(1)
 
-        # On Linux, we check what version of OpenSSL we're working with to figure out
-        # which binary to download.
+        if arch in ["arm64", "aarch64"]:
+            self._write(
+                colorize(
+                    "warning",
+                    "The given CPU architecture ({}) for Linux is in Beta, and only built for one version of Ubuntu-22.04.".format(arch),
+                )
+            )
+            self._write(
+                colorize(
+                    "info",
+                    "Using: Linux-aarch64",
+                )
+            )
+            return "Linux-aarch64"
+
+        # On Linux, determine the distribution and release
+        distro = "Unknown"
+        release = "Unknown"
+
+        # Try to get distribution info from /etc/os-release first
+        self._write(
+                colorize(
+                    "info",
+                    "Checking Linux distribution and version...",
+                )
+            )
+        try:
+            with open("/etc/os-release") as f:
+                os_release = {}
+                for line in f:
+                    line = line.strip()
+                    if line and not line.startswith('#'):
+                        k, v = line.split('=', 1)
+                        os_release[k] = v.strip('"')
+
+                if 'ID' in os_release:
+                    distro = os_release['ID'].capitalize()
+                if 'VERSION_ID' in os_release:
+                    release = os_release['VERSION_ID']
+                self._write(
+                    colorize(
+                        "info",
+                        f"Linux distribution: {distro} Version: {release}",
+                    )
+                )
+        except Exception:
+            self._write(
+                colorize(
+                    "warning",
+                    "Could not determine Linux distribution and version, assuming Ubuntu 22.04",
```

Comment:
Ubuntu 22.04 definitely has /etc/os-release, so the safe thing here is to just exit.